lib/msal-core/package.json

```diff
@@ -48,16 +48,16 @@
     "fork-ts-checker-webpack-plugin": "^0.5.1",
     "husky": "^1.3.1",
     "istanbul-instrumenter-loader": "^3.0.1",
+    "js-yaml": "^3.13.1",
     "jsdom": "14.0.0",
     "jsdom-global": "3.0.2",
     "jshint": "^2.9.7",
-    "karma": "^2.0.5",
+    "karma": "^4.0.1",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-cli": "^1.0.1",
     "karma-coverage-istanbul-reporter": "^2.0.1",
     "karma-firefox-launcher": "^0.1.7",
-    "karma-html-reporter": "^0.2.7",
```

Comment:
We don't need this? Or is this removed to avoid vulnerabilities? Or does js-yaml replace this?

Reply:
un needed.  We don't do any html reporting with karma that I can see.  There are a ton of these we can prune but this one had vulnerabilities

Reply:
Great. Thanks. Yes, we need to plan to prune all unneeded packages eventually.